libkbfs/interfaces.go

```diff
@@ -298,6 +298,9 @@ type KeybaseService interface {
 	// popups spawned.
 	Identify(ctx context.Context, assertion, reason string) (UserInfo, error)
 
+	IdentifyForChat(ctx context.Context, assertion, reason string) (
```

Comment:
I'd prefer a name less chat-specific, since later on we could have other app types that want the same behavior.  Maybe `IdentifyAndAllowTrackBreaks`?